charts/smokey/templates/jobs/frontend-smoke-tests-job.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: frontend-smoke-tests-{{ now | unixEpoch }}
+spec:
+  template:
+    spec:
+      containers:
+      - name: smokey
+        image: "{{ .Values.appImage.repository }}:{{ .Values.appImage.tag }}"
+        command: ["bundle", "exec", "cucumber", "--profile", "test", "--strict-undefined", "-t", "@app-frontend", "-t", "@replatforming", "-t", "not @notreplatforming and not @nottest"]
+        env:
+        - name: ENVIRONMENT
+          value: "{{ .Values.govukEnvironment }}"
+        - name: GOVUK_APP_DOMAIN
+          value: "{{ .Values.appGovukAppDomain | default (printf "%s.%s.%s" .Release.Namespace .Values.govukEnvironment .Values.clusterDomain) }}"
+        - name: GOVUK_WEBSITE_ROOT
+          value: "{{ .Values.appGovukWebsiteRoot | default (printf "https://www-origin-%s.eks.%s.%s" .Release.Namespace .Values.govukEnvironment .Values.govukDomainExternal) }}"
+        # - name: PLEK_SERVICE_FRONTEND_URI
+        #   value: "{{ .Values.appPlekServiceFrontendUri | default (printf "http://frontend.%s.%s" .Release.Namespace .Values.clusterDomain) }}"
```

Comment:
Please either remove the commented-out code or add a TODO saying what needs to happen for it to be uncommented.

Reply:
Good spot